packages/react-magma-dom/src/components/Drawer/Drawer.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { axe } from 'jest-axe';
+import { Drawer } from '.';
+import { render } from '@testing-library/react';
+
+const TEXT = 'Test Text';
+
+describe('Drawer', () => {
```

Comment:
These just look like the default tests, there should probably be more.

Reply:
Added tests to cover the 4 positions